e2ev2/tests/workspace-analysis.test.js

```diff
@@ -24,14 +24,8 @@ browserTest('create an application', async browser => {
   const nextButton = await page.waitForSelector('div[role="button"][aria-label="next"]')
   await expect(nextButton.evaluate(n => n.style.cursor)).resolves.toBe('not-allowed')
 
-  const applicationListDropdown = await page.waitForSelector('input[aria-label="Application List Dropdown"]')
-  /*
-  * Since Prime React uses attaches the aria-label of its Dropdown element to an
-  * invisible element, Puppeteer's click method does not work as intended.
-  * More details can be found here:
-  * https://stackoverflow.com/a/66537619
-  * */
-  await applicationListDropdown.evaluate(b => b.click())
+  const applicationListDropdown = await page.waitForSelector('#application-list-dropdown')
+  await applicationListDropdown.click()
```

Comment:
is this change applicable elsewhere in e2ev2?

Reply:
No, I do not believe so. It seems like the user-create-test was doing something similar. I ended up switching this to the approach that I found there, and it worked. Keeping that change for consistency.